src/Explorer/Tabs/TabsBase.ts

```diff
@@ -40,7 +40,7 @@ export default class TabsBase extends WaitsForTemplateViewModel {
     this.database = options.database;
     this.rid = options.rid || (this.collection && this.collection.rid) || "";
     this.tabKind = options.tabKind;
-    this.tabTitle = ko.observable<string>(options.title);
+    this.tabTitle = ko.observable<string>(this.getTitle() + " - " + options.title);
```

Comment:
`options.title` already contains the collection ID, see https://github.com/Azure/cosmos-explorer/blob/00ffc2d5a950c62d2776ee56d9d0f00376bca95f/src/Explorer/Tree/Collection.ts#L319, so we end up with "[Collection ID] - [Collection ID] - Items" in the title.

Reply:
I do like the arrows.

I believe the ASK was worded a little incorrectly but I went with it. The user wants to be able to identify their tabs easier without having to click on each Collection again to find the one they want.

Reply:
I have reverted the tooltip and corrected the logic.